### PR TITLE
add #3 add hadi chloun

### LINKED ISSUE #3: Add Hadichloun
**Please add your name to the contribution-list.md**

### PR DESCRIPTION
**Pull request by hadichloun**

### CODE REVIEW

File: contribution_list.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # People who have helped!
 
 > Please add your name below to be added to the list of awesome people that have helped with this code.
+
+Hadi Chloun
```
Comment (gleekzorp) [inline contribution_list.md]: Please update to look more like what the others used for styling.

```
- [Genesis Schaerrer](https://github.com/genesisschaerrer)
- [Matthew Ricklefs](https://github.com/matthewricklefs)
- [Ciantera Rose](https://github.com/Ciantera-Rose)
- [Jacob Mason](github.com/JacobMason83)
```

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Please see the comment and fix.

File: contribution_list.md (hunk 1 above)
Comment (hadichloun) [reply contribution_list.md]: ahh.. thats a great idea!

**Review by gleekzorp — APPROVED** [review]
Looks awesome

**Comment by hadichloun** [comment]
awesome! glad i can fix  the changes to your needs and excited to work on project with you!

**Pull request merged**